entity-types/ext-triton/golden_metrics.yml

```diff
@@ -0,0 +1,48 @@
+nv_cpu_memory_total_bytes:
+  title: Provides the total CPU memory from the Triton inference server.
+  unit: BYTES
+  query:
+    select: latest(nv_cpu_memory_total_bytes)
+    from: Metric
+
+nv_cpu_memory_used_bytes:
+  title: Provides the used CPU memory from the Triton inference server.
+  unit: BYTES
+  query:
+    select: latest(nv_cpu_memory_used_bytes)
+    from: Metric
+
+nv_cpu_utilization:
+  title: Provides the CPU utilization from the Triton inference server.
```

Comment:
Is CPU utilization measured in bytes?

Reply:
I updated the unit type to a percentage.